support/cas-server-support-audit-rest/src/main/java/org/apereo/cas/audit/RestAuditTrailManager.java

```diff
@@ -35,10 +30,10 @@
  */
 @Slf4j
 @RequiredArgsConstructor
-public class RestAuditTrailManager implements AuditTrailManager {
+public class RestAuditTrailManager extends AbstractAuditTrailManager {
     private static final ObjectMapper MAPPER = new ObjectMapper()
-        .findAndRegisterModules()
-        .registerModule(new SimpleModule().setMixInAnnotation(AuditActionContext.class, AbstractAuditActionContextMixin.class));
+        .findAndRegisterModules();
+    // .registerModule(new SimpleModule().setMixInAnnotation(AuditActionContext.class, AbstractAuditActionContextMixin.class));
```

Comment:
Is this an accident, or is this line no longer needed?

Reply:
I think it's no longer needed. I added the annotations in Inspektr so the mixin shouldn't be needed anymore.